src/components/BugBountyCards.tsx

```diff
@@ -1,154 +1,105 @@
+    href="https://forms.gle/Gnh4gzGh66Yc3V7G8"
+    {...props}
+  >
     {children}
   </ButtonLink>
 )
 
 const Card = ({ children, ...props }: FlexProps) => {
-  const tableBoxShadow = useToken("colors", "tableBoxShadow")
-
   return (
-    <Flex
-      flexDir="column"
-      flex={{ base: "1 1 412px", xl: "1 1 260px" }}
-      justifyContent="space-between"
-      bg="background.base"
-      borderRadius="2px"
-      borderRad
-      boxShadow={tableBoxShadow}
-      border="1px solid"
-      borderColor="border"
-      m="4"
-      _hover={{
-        borderRadius: "base",
-        boxShadow: "tableBoxHover",
-        background: "tableBackgroundHover",
-        transition: "transform 0.1s",
-        transform: "scale(1.02)",
-      }}
+    <Stack
+      className={cn(
+        "flex-[1_1_412px] gap-0 xl:flex-[1_1_216px]",
+        "m-4 justify-between bg-background",
+        "rounded-sm border border-[#e5e5e5] shadow-table-box dark:border-[#333]",
+        "hover:scale-[1.02] hover:rounded hover:bg-[#f2f2f2] hover:shadow-table-box-hover hover:transition-transform hover:duration-100 dark:hover:bg-[rgba(255,115,36,.013)]"
+      )}
       {...props}
     >
       {children}
-    </Flex>
+    </Stack>
   )
 }
 
 type LabelVariant = "low" | "medium" | "high" | "critical"
 
-type LabelProps = CenterProps & {
+type LabelProps = FlexProps & {
   variant: LabelVariant
 }
 
-const stylePropsByVariant = {
-  low: {
-    bg: "lowBug",
-    color: "black300",
-  },
-  medium: {
-    bg: "mediumBug",
-    color: "black300",
-  },
-  high: {
-    bg: "fail400",
-    color: "white",
-  },
-  critical: {
-    bg: "fail600",
-    color: "white",
-  },
+const classNameByVariant = {
+  low: "bg-[#ffe3d3] text-[#333]",
+  medium: "bg-[#ffab7b] text-[#333]",
+  high: "bg-[#c63333] text-white",
+  critical: "bg-[#930000] text-white",
 }
 
 const Label = ({ children, variant = "medium", ...props }: LabelProps) => {
-  const variantStyleProps = stylePropsByVariant[variant]
+  const variantClassName = classNameByVariant[variant]
 
   return (
     <Center
-      borderTopEndRadius="1px"
-      borderTopStartRadius="1px"
-      borderBottomEndRadius={0}
-      borderBottomStartRadius={0}
-      borderBottom="1px solid"
-      borderColor="border"
-      fontSize="sm"
-      px="0"
-      py="1"
-      textTransform="uppercase"
-      {...variantStyleProps}
+      className={cn(
+        "rounded-t-[1px] border-b border-[#e5e5e5] px-0 py-1 text-sm uppercase",
+        variantClassName
+      )}
       {...props}
     >
       {children}
     </Center>
   )
 }
 
-const H2 = ({ children, ...props }: HeadingProps) => (
-  <Heading
-    fontSize="2xl"
-    fontStyle="normal"
-    fontWeight="bold"
-    lineHeight="22px"
-    letterSpacing="normal"
-    p="4"
-    textAlign="start"
-    mb="-2"
-    mt="2"
+const H2 = ({ children, ...props }) => (
+  <h2
+    className={cn(
+      "-mb-2 mt-2 p-4",
+      "text-start text-2xl font-bold leading-[22px]"
```

Comment:
Just to avoid using custom values
```suggestion
      "text-start text-2xl font-bold leading-6"
```